crates/red_knot/src/module.rs

```diff
@@ -932,7 +932,7 @@ mod tests {
         // Luckily this crate will fail to build if this file isn't available at build time.
         const TYPESHED_ZIP_BYTES: &[u8] =
             include_bytes!(concat!(env!("OUT_DIR"), "/zipped_typeshed.zip"));
-        assert!(!TYPESHED_ZIP_BYTES.is_empty());
```

Comment:
I had to remove this because Rust can now prove that this is always true, lol. I think the remainder of the test still captures the assertions well enough.

Reply:
😭